pkg/operation/seed/seed.go

```diff
@@ -383,16 +383,33 @@ func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubern
 
 	if loggingEnabled {
 		lokiValues["authEnabled"] = false
-		lokiValues["hvpa"] = map[string]interface{}{
-			"enabled": hvpaEnabled,
-		}
 
 		lokiVpa := &autoscalingv1beta2.VerticalPodAutoscaler{ObjectMeta: metav1.ObjectMeta{Name: "loki-vpa", Namespace: v1beta1constants.GardenNamespace}}
 		if err := k8sSeedClient.Client().Delete(ctx, lokiVpa); client.IgnoreNotFound(err) != nil && !meta.IsNoMatchError(err) {
 			return err
 		}
 
 		if hvpaEnabled {
+			shootInfo := &corev1.ConfigMap{}
+			maintenanceBegin := "220000-0000"
+			maintenanceEnd := "230000-0000"
+			if err := k8sSeedClient.Client().Get(ctx, kutil.Key(metav1.NamespaceSystem, "shoot-info"), shootInfo); err != nil {
```

Comment:
If this is for the shooted seed case, shouldn't we use the shoot client instead of the seed client?

Reply:
I am totally confused 😅  This is the bootstrapping case, so `k8sSeedClient` seems right. I take back the comment.

Reply:
The hardcoded maintenance window is when the `shoot-info` CM is not found in the seed cluster. In this case I think that it does not matter if the maintenance window is hardcoded or random-generated.

The maintenance window is added only for Loki in `garden` namespace in the seed cluster, e.g. `ShootedSeed` cluster.
We also have Lokis in the shoot-namespaces but for now I think that it will be better to not add the maintenance window on them.